docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1270 @@
+    its associated schema file.</para>
+
+    <sect2 id="WhatIsValues.Yaml">
+      <title>What is values.yaml and how is it used?</title>
+
+      <para>The <emphasis>values.yaml</emphasis> file is the delivered
+      configuration values for a Helm chart. The values.yaml file is used by
+      the Helm chart to control how HPCC Systems is deployed to the cloud.
+      This values file is one file used to configure and get an HPCC Systems
+      instance up and running on Kubernetes. The values.yaml file defines
+      everything that happens to configure and/or define your system for a
+      containerized deployment. You should use the values file as a basis for
+      modeling the specific customizations for your deployment specific to
+      your requirements.</para>
+
+      <para>The HPCC Systems values.yaml file can be found in the HPCC Systems
+      github repository. To use the HPCC Systems Helm chart, first add the
+      hpcc chart repository using Helm, then access the Helm chart values from
+      the charts in that repository.</para>
+
+      <para>For example, when you add the "hpcc" repository, as recommended
+      prior to installing the Helm chart with the following command:</para>
+
+      <programlisting>helm repo add hpcc https://hpcc-systems.github.io/helm-chart
+</programlisting>
+
+      <para>You can now view the HPCC Systems delivered charts and see the
+      values there by issuing:</para>
+
+      <programlisting>helm show values hpcc/hpcc</programlisting>
+
+      <para>You can capture the output of this command, look at how the
+      defaults are configured and use it as a basis for your
+      customization.</para>
+    </sect2>
+
+    <sect2 id="Values-SchemaJSONFile">
+      <title>The values-schema.json</title>
+
+      <para>The <emphasis>values-schema.json</emphasis> is a JSON file that
+      declares what is valid and what is not within the sum total of the
+      merged values that are passed into Helm at install time. It defines what
+      values are allowed, and validates the values file against them. All the
+      core items are declared in the schema file, while the default
+      values.yaml file also contains comments on the most important elements.
+      If you wanted to know what options are available for any particular
+      component then the schema is a good place to start.</para>
+
+      <para>The schema file typically contains (for a property) a name and a
+      description. It will often include details of the type, and the items it
+      can contain if it is a list or dictionary. For instance:</para>
+
+      <programlisting>    "roxie": { 
+      "description": "roxie process",
+      "type": "array"
+      "items": { "$ref": "#/definitions/roxie" }
+    },</programlisting>
+
+      <para>The "items" in this case are a reference ($ref) to another section
+      of the schema. Each plane, in the schema file has a list of properties
```

Comment:
This seems to be a mixture of an example and a general statement.  I think clearer would be to move this after the next statement, and start it with

"For example, for a storage plane the schema file has a list of properties including a prefix...."